src/api/error.js

```diff
@@ -26,6 +26,17 @@ Object.defineProperty( INatApiError.prototype, "name", {
   value: "INatApiError"
 } );
 
+export class INatApiUnauthorizedError extends INatApiError {
+  constructor( context?: Object ) {
+    const errorJson = {
+      error: "Unauthorized",
+      status: 401,
+      context
+    };
+    super( errorJson, 401, context );
+  }
+}
+
```

Comment:
Does this need `Object.defineProperty( INatApiUnauthorizedError.prototype, "name", {
  value: "INatApiUnauthorizedError"
} );`? The other two errors have that defined.

Reply:
Have added